sourced/ml/tests/test_bblfsh_utils.py

```diff
@@ -0,0 +1,77 @@
+import docker.client
+import random
+import time
+import unittest
+
+from sourced.ml.utils.bblfsh import check_version, BBLFSH_VERSION_LOW, BBLFSH_VERSION_HIGH
+
+
+def check_bblfsh_version_support(client: docker.client.DockerClient, version: str) -> bool:
+    """
+    :param client: docker client instance
+    :param version: version of bblfshd to check
+    :return: True if version is supported, False otherwise
+    """
+    for i in range(3):
```

Comment:
This seems a bit hacky. Why three times? If you need to wait until the port becomes open, use a port check with `socket.connect`.

Reply:
I'm picking a random port, something might be running there already.

Reply:
You can check this using socket.connect